packages/hotjar/README.rst

```diff
@@ -0,0 +1,15 @@
+@edx/frontend-enterprise-hotjar
+=============
+
+|Build Status| |npm_version| |npm_downloads| |license|
+
+This package exports useful React hooks, utility functions, and constants for using Hotjar.
+
+.. |Build Status| image:: https://github.com/edx/frontend-enterprise/actions/workflows/release.yml/badge.svg
+   :target: https://github.com/edx/frontend-enterprise/actions
+.. |npm_version| image:: https://img.shields.io/npm/v/@edx/frontend-enterprise-utils.svg
+   :target: @edx/frontend-enterprise-utils
```

Comment:
nit: `@edx/frontend-enterprise-utils`  -> `@edx/frontend-enterprise-hotjar` throughout this file